src/modules/platform-tools/terminal/terminal.service.ts

```diff
@@ -17,12 +18,33 @@ export interface TermSize {
 @Injectable()
 export class TerminalService {
   private ending = false
+  private static persistentTerminal: any = null
```

Comment:
is there a better type for this? to be honest i assume if there was, then you wouldn't have used any 😀